temporian/core/operators/window/test/test_moving_sum.py

```diff
@@ -197,6 +197,14 @@ def test_with_variable_winlen_same_sampling_uses_correct_cpp_impl(
             window_length=window_length.data[()].features[0],
         )
 
+    def test_error_input_bytes(self):
+        evset = event_set([1, 2], {"f": ["A", "B"]})
+        with self.assertRaisesRegex(
+            ValueError,
+            "moving_sum requires numerical point inputs",
+        ):
+            _ = evset.moving_sum(1)
```

Comment:
missing test for moving_std, moving_min, moving_max (we will migrate the old tests soon, but we might forget to add these if they aren't there already)

Reply:
Good point.
*Adding min/max/sd tests